### DISCUSSION TITLE
Print Error trigger fires on every print

### DISCUSSION THREAD

**Discussion by LightDemon** [greghesp/ha-bambulab#403] [Q&A]
Has anyone else seen an issue on the X1C where the print error detected trigger fires on every print with the message of inspecting first layer, As far as I can tell there is no error reported on the printer so it seems like the fact that it is inspecting the first layer is being reported as a print error.

**Comment by AdrianGarside** [comment]
Yes? I'm pretty consistently but not 100% consistently getting an error that shows in bambu handy as well. They seem to be firing an 'error' that the filament is transmissivity isn't good enough for accurate first layer issue detection a lot more with the recent firmware. Initially I thought it was because I was printing in black but I got it on more recent prints with white pla too. Unless you're seeing something different?
